src/enum/appointment.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum AppointmentStatus {
+    Pending = 'pending',
+    Approved = 'approved',
+    Cancelled = 'cancelled',
+    Finished = 'finished',
+}
```

Comment:
According to previous comment about redundant API endpoint. We should consider add `rejected` so that we can separate the situation to _cancelled by student_ OR _rejected by tutor_